### PR TITLE
[ResourceDetectors.Azure] Update CHANGELOG.md - Request for release

### PR DESCRIPTION
**Pull request by vishweshbankwar**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # Changelog
 
-## Unreleased
+## 1.0.0-beta.3
 
+Released 2023-Sep-19
+
+* Added a timeout for `HttpClient` for outgoing call made to metadata service to
```
Comment (cijothomas) [inline src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md:7]: please add the timeout value here itself, so users does not have to open the PR and then the code in it to understand the actual value.

File: src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+  improve start up time of application.
```
Comment (cijothomas) [inline src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md:8]: improvement is mostly only if running in non-azure vm?

Comment (vishweshbankwar) [reply src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md]: clarified

File: src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md (hunk 1 above)
Comment (vishweshbankwar) [reply src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md]: done

File: src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,13 @@
 # Changelog
 
-## Unreleased
+## 1.0.0-beta.3
 
+Released 2023-Sep-19
+
+* Added a timeout of `2 seconds` for `HttpClient` for outgoing call made to
```
Comment (utpilla) [inline src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md:7]: Rephrasing suggestion:

```md
* Configured the `HttpClient` used for making the call to metadata service to use a `Timeout` to `2` seconds. This is to improve the start-up time of applications not running on Azure VMs.
```

Comment (vishweshbankwar) [reply src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md]: done

File: src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
 
-## Unreleased
+## 1.0.0-beta.3
 
+Released 2023-Sep-19
+
+* Configured the `HttpClient` used for making the call to metadata service to
+  use a `Timeout` to `2` seconds. This is to improve the start-up time of
+  applications not running on Azure VMs.
+  ([#1358](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1358))
```
Comment (utpilla) [inline src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md:10]: I think you added the PR link twice.

Comment (vishweshbankwar) [reply src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md:10]: oh no, sorry about that. fixed.

File: src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md [hunk 5]
```diff
@@ -1,7 +1,13 @@
 # Changelog
 
-## Unreleased
+## 1.0.0-beta.3
```
Comment (Kielek) [inline src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md:3]:
```suggestion
## Unreleased

## 1.0.0-beta.3
```

**Review by Kielek — APPROVED** [review]

**Comment by Kielek** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/releases/tag/ResourceDetectors.Azure-1.0.0-beta.3

**Pull request merged**